openshift-knative-operator/cmd/operator/kodata/knative-serving/0.19.0/2-serving-core.yaml

```diff
@@ -248,7 +248,33 @@ roleRef:
   kind: ClusterRole
   name: knative-serving-admin
   apiGroup: rbac.authorization.k8s.io
-
+---
```

Comment:
Can we put these in separate files? (Along with the YAML above, that can move into the "manifest" itself too I believe)

Reply:
ok